resources/public/api/conf/1.0/application.yaml

```diff
@@ -30,6 +30,17 @@ components:
           scopes:
             write:self-assessment: Grant write access
             read:self-assessment: Grant read access
+    Application-Restricted:
+      type: oauth2
+      description: |
+        HMRC supports OAuth 2.0 for authenticating application restricted API requests using an OAuth 2.0 Bearer Token
+        in the AUTHORIZATION header. See https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation/application-restricted-endpoints for details.
+      flows:
+        clientCredentials:
+          tokenUrl: https://api.service.hmrc.gov.uk/oauth/token
+          scopes:
```

Comment:
Not sure on the scopes here? I copied them over from User-Restricted above, I assume both still apply?